src/lib.rs

```diff
@@ -84,6 +86,12 @@ pub mod usb;
 #[cfg(feature = "device-selected")]
 pub mod watchdog;
 
+// TODO other controllers have CAN
+#[cfg(any(
+    feature = "stm32f091",
```

Comment:
```suggestion
    feature = "stm32f091",
    feature = "stm32f042",
    feature = "stm32f072",
```

That's all of them according to the ST website.